Source/Extensions/Blazorise.Splitter/SplitterSection.razor.cs

```diff
@@ -0,0 +1,109 @@
+﻿#region Using directives
+using System;
+using System.Threading.Tasks;
+using Blazorise.Extensions;
+using Blazorise.Utilities;
+using Microsoft.AspNetCore.Components;
+#endregion
+
+namespace Blazorise.Splitter;
+
+/// <summary>
+/// A resizable section of a <see cref="Splitter"/> component.
+/// </summary>
+public partial class SplitterSection : BaseComponent, IDisposable
+{
+    #region Methods
+
+    public override Task SetParametersAsync( ParameterView parameters )
+    {
+        if ( Rendered )
+        {
+            var sizeChanged = parameters.TryGetValue<double?>( nameof( Size ), out var paramSize ) && !Size.IsEqual( paramSize );
+            var minSizeChanged = parameters.TryGetValue<double>( nameof( MinSize ), out var paramMinSize ) && !MinSize.IsEqual( paramMinSize );
+            var maxSizeChanged = parameters.TryGetValue<double>( nameof( MaxSize ), out var paramMaxSize ) && !MaxSize.IsEqual( paramMaxSize );
+            var snapOffsetChanged = parameters.TryGetValue<double>( nameof( SnapOffset ), out var paramSnapOffset ) && !SnapOffset.IsEqual( paramSnapOffset );
+
+            if ( sizeChanged
+                || minSizeChanged
+                || maxSizeChanged
+                || snapOffsetChanged )
+            {
+                Parent?.UpdateSection( this );
+            }
+        }
+
+        return base.SetParametersAsync( parameters );
+    }
+
+    /// <inheritdoc />
+    protected override void OnInitialized()
+    {
+        if ( Parent is null )
+            throw new ArgumentNullException( nameof( Parent ), nameof( SplitterSection ) + " must exist within a Split" );
```

Comment:
"must exist within a Split"; you mean must exist within Splitter? use **nameof**